Control/Metrics/src/Dockerfile

```diff
@@ -8,6 +8,8 @@ WORKDIR /src
 COPY ["Control/Metrics/src/ArmoniK.Core.Control.Metrics.csproj", "Control/Metrics/src/"]
 COPY ["Adaptors/Redis/src/ArmoniK.Core.Adapters.Redis.csproj", "Adaptors/Redis/src/"]
 COPY ["Common/src/ArmoniK.Core.Common.csproj", "Common/src/"]
+COPY ["Utils/src/ArmoniK.Core.Utils.csproj", "Utils/src/"]
+COPY ["Base/src/ArmoniK.Core.Base.csproj", "Base/src/"]
```

Comment:
I think it can be omitted

Reply:
We need it for Common otherwise there are warnings during build